CI_BUILDERS.md

```diff
@@ -0,0 +1,29 @@
+# Documentation for CI Builders
+
+Members of the ros2/ci-builders team on GitHub.com can trigger builds on <https://ci.ros2.org> when signed in via GitHub.com.
+This team is primarily composed of collaborators from other projects that work closely with the ROS 2 core so they can verify their changes remain compatible with ROS 2.
+
+
+## Testing pull requests
+
+For pull requests, use the [CI launcher job](https://ci.ros2.org/job/ci_launcher) to trigger builds against Ubuntu AMD64, Ubuntu ARM64, macOS, and Windows.
+
+When changes are part of the default repository use the `CI_BRANCH_TO_TEST` parameter to checkout and merge that branch against the default branch for all repositories with a matching branch name.
+This works well for testing changes in multiple repositories together.
+Many of the other arguments should be reasonably self-explanatory and its likely you won't change them from their defaults to do Pull request CI, although you will want to be sure that the CI_USE_CYCLONEDDS box is checked. CI_BUILD_ARGS and CI_TEST_ARGS are basically passed directly to colcon build and test respectively so you should be familiar with those.
```

Comment:
nitpick: One sentence per line (makes for nicer diffs in the future).

Also, we can style the vars for consistency.
```suggestion
Many of the other arguments should be reasonably self-explanatory and its likely you won't change them from their defaults to do Pull request CI, although you will want to be sure that the `CI_USE_CYCLONEDDS` box is checked.
`CI_BUILD_ARGS` and `CI_TEST_ARGS` are basically passed directly to colcon build and test respectively so you should be familiar with those.
```